packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -386,33 +386,59 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                 )}
 
                 {allowTimepointProperties && showLinkToStudy && (
-                    <Row className="margin-top">
-                        <Col xs={2}>
-                            <DomainFieldLabel
-                                label="Auto-Link Data to Study"
-                                helpTipBody={
-                                    <>
-                                        <p>
-                                            Automatically link Sample Type data rows to the specified target study. Only
-                                            rows that include subject and visit/date information will be linked.
-                                        </p>
-                                        <p>
-                                            The user performing the import must have insert permission in the target
-                                            study and the corresponding dataset.
-                                        </p>
-                                    </>
-                                }
-                            />
-                        </Col>
-                        <Col xs={5}>
-                            <AutoLinkToStudyDropdown
-                                containers={containers}
-                                onChange={this.onFormChange}
-                                autoLinkTarget={ENTITY_FORM_IDS.AUTO_LINK_TARGET}
-                                value={model.autoLinkTargetContainerId}
-                            />
-                        </Col>
-                    </Row>
+                    <>
```

Comment:
I waffled about modularizing the section below (389-441) into a component called AutoLinkSettings or such. 

Pros: It's long, and tucking it away somewhere might be good, especially as most people working on/looking at SampleTypePropertiesPanel probably won't be dealing with the link to study settings
Cons: The other settings options are not modularized, and it is pretty readable and clear to just scan through and see what options are surfacing under what conditions. Modularizing this away would be a lil departure from that pattern

Reply:
I think it is fine to leave these in-line here. One thing that might help though would be to move the `helpTipBody` values to constants as those are rather long and not likely to change so moving them out might help the readability.